index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">
+    <div class="wrapper">
       <div class="nav-left">
-        <a href="#" class="logo">BankPL</a>
-        <p class="offer-name">Konto 360°</p>
+        <a href="#" class="site-logo">BankPL</a>
+        <span class="offer-name">Konto 360°</p>
       </div>
-      <a href="#" class="nav-icon">&#9776;</a>
+      
+      <button href="#" class="nav-icon">&#9776;</button>
+
       <ul class="main-nav">
-        <li><a href="#">O koncie</a></li>
-        <li><a href="#">Pakiet Bardzo Pomocny</a></li>
-        <li><a href="#">Lubię to polecam</a></li>
-        <li><a href="#">Jak otworzyć</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">O koncie</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Pakiet Bardzo Pomocny</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Lubię to polecam</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Jak otworzyć</a></li>
       </ul>
-    </nav>
-    <div class="header-title">
-      <h1>Konto 360° pełne korzyści</h1>
-      <p>z aplikacją mobilną na każdą pogodę</p>
-      <button class="btn-top">Załóż konto 360° &gt;</button>
-    </div>
-  </header>
-  <article id="banner Top">
-    <div class="bar-one">
-      <h1>O koncie</h1>
     </div>
-    <p class="simple-place"><strong>Korzystaj z konta za 0 zł</strong> w dowolnym miejscu i czasie</p>
-    <div class="banner-one">
-      <h2>0zł</h2>
-      <p><b>za prowadzenie konta, kartę i wypłaty ze wszystkich bankomatów,</b></br> jeśli aktywnie korzystasz z konta osobistego, czyli przelewasz na nie swój dochód (min. 1000zł wpływów zewnętrznych miesięcznie) i </br> płacisz min. raz w miesiącu kartą do konta lub BLIKIEM</p>
+  </nav>
+
+  <header class="hero__container">
+    <div class="wrapper hero__content">
+      <h1 class="hero__title">Konto 360° pełne korzyści</h1>
+      <span class="hero__hint">z aplikacją mobilną na każdą pogodę</span>
+      <button class="btn btn--hero hero__btn">Załóż konto 360° &gt;</button>

```

Comment:
tu bylo `.btn-top`:
- dodalem klase ogolna `.btn` - dla wszystkich przyciskow na stornie, 
- klase `btn--hero` - zeby stylowac tylko ten konkretny przycisk 
- klase `hero__btn` - to samo co wyzej, kwestia podejscia i jak bardzo ten przycisk rozni sie od ogolnego

generalnie nie zalecane jest uzywac obydwu klas tutaj
 - powinno sie wybrac czy jako baze bierzemy ze jest to przycisk - i jego inna wersje - `btn--hero` 
- czy jako baze bierzemy ze jest to w sekcji hero, i jakis tam w niej przycisk - `hero__btn` 

niby nic waznego - ale warto sie zastanowic czy dany wyglad przycisku sie powtorzy w innym miejscu niz w tym hero, i na podstawie tego okreslic - jesli sie powtorzy to jest to `btn--hero`, a jesli sie nie powtorzy to `hero__btn` :)